stablehlo/tests/stablehlo_legalize_quant_to_int.mlir

```diff
@@ -0,0 +1,2229 @@
+// RUN:  stablehlo-opt --stablehlo-legalize-quant-to-int -split-input-file %s -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL: func @uniform_quantize_and_dequantize
+func.func @uniform_quantize_and_dequantize(%arg0: tensor<?x?xf32>) -> tensor<?x?xf32> {
+  // CHECK-DAG: %[[SCALES:.*]] = stablehlo.constant dense<1.000000e+00> : tensor<f32>
```

Comment:
Also want to confirm prior to approval that we plan to remove the MHLO version of this pass? This is a lot of (hopefully temporary) duplication.

Reply:
Agreed!